test/ILLink.RoslynAnalyzer.Tests/LinkerTestCases.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Xunit;
+
+namespace ILLink.RoslynAnalyzer.Tests
+{
+	/// <summary>
+	/// Test cases stored in files
+	/// </summary>
+	public class LinkerTestCases : TestCaseUtils
+	{
+		[Theory]
```

Comment:
In this case it's filtering out methods since the analyzer doesn't support them. However, it's still somewhat useful to be able to narrow a test down to a small scenario. Also, my goal was to not touch the linker test code at all.